subsys/bluetooth/audio/ascs.c

```diff
@@ -2905,4 +2993,7 @@ void bt_ascs_cleanup(void)
 		unicast_server_cb = NULL;
 	}
 }
+
+K_THREAD_DEFINE(ascs, CONFIG_BT_ASCS_STACK_SIZE, ascs_thread, NULL, NULL, NULL,
```

Comment:
I am wondering if we can get this working without the need for a separate thread (thinking memory constraints)?

Reply:
I decided to make use of the System Workqueue thread